examples/sample-pub/metering.js

```diff
@@ -0,0 +1,41 @@
+/** Helps demonstrate metering functionality. */
+const MeteringDemo = {
+  /** Sets up controls for the metering demo. */
+  setupControls: () => {
+    // Wire up reset button.
+    document
+      .querySelector('#metering-controls button')
+      .addEventListener('click', () => {
+        // Forget the existing PPID.
```

Comment:
Could you spell out what PPID is in one of these comments?

Reply:
Definitely, I spelled it out in a couple of comments below